accounts/abi/bind/base.go

```diff
@@ -431,16 +464,25 @@ func (c *BoundContract) transact(opts *TransactOpts, contract *common.Address, i
 	return signedTx, nil
 }
 
+// FilterLogsByID filters contract logs for past blocks, returning the necessary
+// channels to construct a strongly typed bound iterator on top of them.
+func (c *BoundContract) FilterLogsByID(opts *FilterOpts, eventID common.Hash, query ...[]interface{}) (<-chan types.Log, event.Subscription, error) {
```

Comment:
I will remove this method:
Can opt to just emit the event name as a Go constant in the bindings, then pass that to `BoundContract.FilterLogs` along with the abi (which we already emit).